profiles/common/modules/features/multisite_business_indicators/multisite_business_indicators_core/tests/multisite_business_indicators_core.test

```diff
@@ -9,9 +9,12 @@
  */
 class BusinessIndicatorsCoreQAWebTestCase extends DrupalWebTestCase {
   protected $profile = 'multisite_drupal_communities';
```

Comment:
Same thing here, I wasn't clear. Class documentation was good, but the `$profile` property was missing `{@inheritdoc}`. My apologies!